src/components/tabs/index.js

```diff
@@ -97,10 +94,9 @@ export default class ButterTabs extends Component {
                         <ModalContent/>
                     </Modal>
                 </CSSTransitionGroup>
-                <NavBar {...props} selected={state.selected} action={{
-                    toggleAdvanced: this.toggleAdvanced.bind(this),
-                    goBack: props.action.goBack
-                }}/>
+
+                <Navbar {...props.navbar}/>
```

Comment:
this changes the API of NavBar, are we sure we want that ? it's ok but it will need changes in `-desktop` too